src/services/tasks/index.ts

```diff
@@ -1,15 +1,40 @@
-import type { HttpInstance } from '../../types';
+import { ApiRequestService } from '../../types';
+import type {
+  ApiService,
+  FunctionsService,
+  HttpInstance,
+  LogsService,
+  SchedulesService,
+} from '../../types';
 import httpClient from '../http-client';
+import api from './api';
+import apiRequests from './apiRequests';
+import functions from './functions';
+import logs from './logs';
+import schedules from './schedules';
 import tasks from './tasks';
 import { TASKS_BASE } from '../../constants';
 import { TasksService } from './types';
 
-export const tasksService = (httpWithAuth: HttpInstance): TasksService => {
```

Comment:
Moved all of it here from the tasks functions. Probably should not have done it in this PR 😅

Reply:
It was a bit confusing, but nice change though 🎉